docs/products/grafana/howto/grafana-authentication-plugins.rst

```diff
@@ -0,0 +1,106 @@
+Grafana® authentication plugins
+##############################
+
+Configure Google, GitHub or GitLab authentication integration to Aiven for Grafana®.
+-------------------------------
+
+Grafana supports multiple different authentication plugins, in addition to built-in username and password authentication.
+
+Aiven Grafana has support for Google, GitHub and GitLab authentication. Configuring authentication integration is currently available in Aiven console, or can be configured either with the API or with `Aiven command-line client <https://github.com/aiven/aiven-client>`_ (``avn``).
+
+Please note that enabling this feature requires applying a maintenance upgrade for Grafana services created before September 2018.

```

Comment:
Do we need this if it's for services nearly 4 years old? Maybe we can remove this paragraph.